ui/src/Elements/Message/Message.assets.js

```diff
@@ -1,5 +0,0 @@
-import { CONSTANTS } from 'Utils';
```

Comment:
Out of interest, what was your rationale for removing the assets? I would prefer that it stay as it buys us some abstraction (we could change where the constants come from without changing the rest of the component) and reuse across things like storybook and tests (eg, the `defaultMessages` I see in your storybook file could be used there and for tests (saving the duplication for `testMessage` in your spec file) say).